packages/miniflare/src/plugins/core/modules.ts

```diff
@@ -30,9 +30,9 @@ const builtinModulesWithPrefix = builtinModules.concat(
 
 // Module identifier used if script came from `script` option
 export function buildStringScriptPath(workerIndex: number) {
-	return `script:${workerIndex}`;
+	return `script-${workerIndex}`;
```

Comment:
The colon here cause issue with how stack trace is parsed by both `youch` and `@cspotcode/source-map-support` as `:` is used to indicate column and row.